lib/include/kickcat/Mailbox.h

```diff
@@ -199,8 +202,9 @@ namespace kickcat::mailbox::response
         void replyError(std::vector<uint8_t>&& raw_message, uint16_t code);
 
         AbstractESC* esc_;
-        SyncManagerConfig mbx_in_;
-        SyncManagerConfig mbx_out_;
+        std::optional<SyncManagerConfig> mbx_in_;
```

Comment:
I would rather like to use the field type (SyncManagerType=) == unused instead of std::optional since theses members are not optional to have a working mailbox.